src/main/webapp/WEB-INF/tags/student/registerMessage.tag

```diff
@@ -0,0 +1,10 @@
+<%@ tag description="Register message for unregistered students" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ attribute name="googleId" required="true" %>
+<%@ attribute name="registerMessage" required="true" %>
+
+<c:if test="${empty googleId}">
+    <div id="registerMessage" class="alert alert-info">
+        ${registerMessage}
+    </div>
+</c:if>
```

Comment:
Is this used in multiple places? Which pages use it?
I don't see much of a need to extract something this short otherwise.

(might be good to mention them in the description if multiple pages use it)